src/psyclone/tests/psyir/symbols/symbol_table_test.py

```diff
@@ -1978,6 +1978,14 @@ def test_rename_symbol_errors():
     assert ("Cannot rename symbol 'frankie' because it is a routine argument "
             "and as such may be named in a Call." in str(err.value))
 
+    # Cannot rename a common block symbol
```

Comment:
Not yours but please could you fix the pylint warning about unused `var2` at L1324?

Reply:
Done